app/src/cambodia/java/org/eyeseetea/malariacare/domain/usecase/LoginUseCase.java

```diff
@@ -1,41 +0,0 @@
-package org.eyeseetea.malariacare.domain.usecase;
-
-import android.content.Context;
-
-import org.eyeseetea.malariacare.LoginActivity;
-import org.eyeseetea.malariacare.R;
-import org.eyeseetea.malariacare.data.database.model.User;
-import org.eyeseetea.malariacare.data.database.utils.PreferencesState;
-import org.eyeseetea.malariacare.data.database.utils.Session;
-import org.eyeseetea.malariacare.domain.entity.Credentials;
-import org.eyeseetea.malariacare.network.ServerAPIController;
-
-public class LoginUseCase extends ALoginUseCase {
```

Comment:
@idelcano I think that LoginUseCase should exists per variant, because and lao and cambodia has a different behavior. In Myanmar login is standard login but in lao and cambodia should login with user credentials but when login is realized than execute authenticationManager.Logout and AuthenticationManager.Login with default credentials user:KHMCS pwd: =KHMCSadmin1. Default credentials should be parameters of execute method in use case.